diagnostic.md

```diff
@@ -57,13 +65,17 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li').on( function(){
```

Comment:
Adding an event handler here is unnecessary since we're just traversing nodes and the call to `.on()` is invalid since the first argument needs to be the name of an event in the form of a string.  One possible way is `$('li').parent().parent().hide();`.